[OSDOCS-2719] Adding the web console overview and Administrator perspective (#47809)
openshift/openshift-docs: web_console/web-console-overview.adoc

:_content-type: ASSEMBLY
[id="web-console-overview"]
= Web Console Overview
include::_attributes/common-attributes.adoc[]
:context: web-console-overview

toc::[]

> The Red Hat {product-title} web console provides a graphical user interface to visualize your project data and perform administrative, management, and troubleshooting tasks. The web console runs as pods on the control plane nodes in the openshift-console project. It is managed by an operator pod. Both *Administrator* and *Developer* perspectives are supported.

Reviewer: ```suggestion
The Red Hat {product-title} web console provides a graphical user interface to visualize your project data and perform administrative, management, and troubleshooting tasks. The web console runs as pods on the control plane nodes in the openshift-console project. It is managed by a console-operator pod. Both *Administrator* and *Developer* perspectives are supported.
```

> Both *Administrator* and *Developer* perspectives enable you to create quick start tutorials for the OpenShift Container Platform. A quick start is a guided tutorial with user tasks and is useful for getting oriented with an application, operator, or other product offering.

Reviewer 2: When you say "create quick start," do you mean actually creating a new quick start or just running through one? We might rephrase if the latter.

Reviewer 3: It looks like creating a new one based on what we have in the docs https://docs.openshift.com/container-platform/4.10/web_console/creating-quick-start-tutorials.html

:_content-type: ASSEMBLY
[id="web-console-overview"]
= Web Console Overview
include::_attributes/common-attributes.adoc[]
:context: web-console-overview

toc::[]

> The Red Hat {product-title} web console provides a graphical user interface to visualize your project data and perform administrative, management, and troubleshooting tasks. The web console supports both *Administrator* and *Developer* perspectives.

Reviewer 3: WDYT of combining these?

```suggestion
The Red Hat {product-title} web console provides a graphical user interface to visualize your project data and perform administrative, management, and troubleshooting tasks. The web console runs as pods on the control plane nodes in the openshift-console project. It is managed by an operator pod. Both *Administrator* and *Developer* perspectives are supported.
```

> The web console runs as pods on the control plane nodes in the openshift-console project. It is managed by an operator pod

Reviewer 3: ```suggestion
The web console runs as pods on the control plane nodes in the openshift-console project. It is managed by an operator pod.
```

Both *Administrator* and *Developer* perspectives enable you to create quick start tutorials for the OpenShift Container Platform. A quick start is a guided tutorial with user tasks and is useful for getting oriented with an application, operator, or other product offering.

> 

Reviewer 3: remove extra line

include::modules/about-administrator-perspective.adoc[leveloffset=+1]
include::modules/accessing-the-administrator-perspective.adoc[leveloffset=+2]

> 

Reviewer 3: remove extra line

include::modules/about-developer-perspective.adoc[leveloffset=+1]
include::modules/odc-accessing-developer-perspective.adoc[leveloffset=+2]

> 

Reviewer 3: remove extra line

:_content-type: ASSEMBLY
[id="web-console-overview"]
= Web Console Overview
include::_attributes/common-attributes.adoc[]
:context: web-console-overview

toc::[]

The Red Hat {product-title} web console provides a graphical user interface to visualize your project data and perform administrative, management, and troubleshooting tasks. The web console runs as pods on the control plane nodes in the openshift-console project. It is managed by a `console-operator` pod. Both *Administrator* and *Developer* perspectives are supported.

> Both *Administrator* and *Developer* perspectives enable you to create quick start tutorials for the OpenShift Container Platform. A quick start is a guided tutorial with user tasks and is useful for getting oriented with an application, operator, or other product offering.

Reviewer 4: s/the OpenShift Container Platform/{product-title}

s/operator/Operator

[role="_additional-resources"]
.Additional resources

* xref:../welcome/learn_more_about_openshift.adoc#cluster-administrator[Learn more about Cluster Administrator]
* xref:../getting_started/openshift-overview.adoc#for-administrators[Overview of the *Administrator* perspective]
* xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Creating and deploying applications on {product-title} using the *Developer* perspective]
* xref:../applications/odc-viewing-application-composition-using-topology-view.adoc#odc-viewing-application-composition-using-topology-view[Viewing the applications in your project, verifying their deployment status, and interacting with them in the *Topology* view]
* xref:../web_console/using-dashboard-to-get-cluster-information.adoc#using-dashboard-to-get-cluster-info[Viewing cluster information]
* xref:../web_console/configuring-web-console.adoc#configuring-web-console[Configuring the web console]
* xref:../web_console/customizing-the-web-console.adoc#customizing-web-console[Customizing the web console]

> * xref:../web_console/odc-about-web-terminal.adoc#odc-about-web-terminal[Launching an embedded command line terminal inst in the web console]

Reviewer 4: s/inst/instance
